server.js

```diff
@@ -3,21 +3,85 @@ const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
+const bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({ extended: true }));
 
-const welcomeMessage = {
+app.use(cors());
+app.use(express.json());
+const welcomeMessage = 
+{
   id: 0,
   from: "Bart",
   text: "Welcome to CYF chat system!",
-};
+}
+
+  
 
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
+// get html page
+app.get("/", function (req, res) {
+  res.sendFile(__dirname + "/index.html");
```

Comment:
This is fine for this example, but out of interest you can also ask express to serve a whole directory of static files rather than just individual ones- less verbose once you have a few of them:
https://www.digitalocean.com/community/tutorials/nodejs-serving-static-files-in-express